common/src/main/java/com/mx/path/core/common/connect/AccessorConnectionSettings.java

```diff
@@ -20,12 +21,14 @@
 @AllArgsConstructor
 public class AccessorConnectionSettings implements ConnectionSettings {
 
+  private List<RequestFilter> baseRequestFilters;
   private String baseUrl;
   private String certificateAlias;
   private ObjectMap configurations;
+  private Duration connectTimeOut;
   private char[] keystorePassword;
   private String keystorePath;
-  private List<RequestFilter> baseRequestFilters;
+  private Duration requestTimeOut;
```

Comment:
'timeout' is one word. This shouldn't be capitalized as "requestTimeOut". It should be "requestTimeout".

Reply:
This same comment throughout the MR.

Reply:
Pushed fixes